### DISCUSSION TITLE
Problem with pairing/configuration of HB devices on RaspberryMatic > 3.55.10.20210213

### DISCUSSION THREAD

**Discussion by soosp** [OpenCCU/OpenCCU#1293] [🇺🇸 Q&A] [answered]
Hi All,

I'm running a little HomeMatic system with some HomeMatic, HomeMatic IP and HomeBrew devices. I use an original CCU3 hardware updated to RaspberryMatic. It worked fine before updated to releases > 3.55.10.20210213. After updating I'm unable to pair with or set configuration of HB devices. After downgraded to 3.55.10.20210213 all of them works well again. I'm speaking English (or Hungarian :) ) only, so please write in English if possible.

**Comment by Baxxy13** [comment, marked as answer]
Hi,
RaspberryMatic and CCU3 Firmwares > 3.55.10 comes with a new RF-Module Firmware. These new Firmware slightly modifies the frequency the RF-Module uses. If your problem is only related to your HB-Devices you hava to flash the so called [FreqTest-Sketch](https://asksinpp.de/Grundlagen/FAQ/Fehlerhafte_CC1101.html#ermittlung-der-cc1101-frequenz). After success you have to re-flash your original Sketch. 

For help with that i recommend asking in the [englisch section of the Homematic-Forum](https://homematic-forum.de/forum/viewforum.php?f=49).  There are the guys with the knowledge and they can hopefully write and help in english.

**Reply by soosp** [reply]
Sorry for late answare. This was the solution :)
Thanks!
